homework/css/components/main.css

```diff
@@ -0,0 +1,107 @@
+main {
+  grid-area: main;
+  color: black;
+  background-color: #f4fff8;
+}
+
+main section {
+  display: grid;
+  grid-gap: 5px;
+}
+
+main span {
+  display: block;
+}
+
+main article {
+  min-width: 200px;
```

Comment:
width and height are 100% by default. You could remove these styles

Reply:
Done